esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -1681,27 +1726,21 @@ void CTpWrapper::getTpDropZones(double clientVersion, const char* name, bool ECL
     {
         IPropertyTree & plane = planes->query();
         const char * dropzonename = plane.queryProp("@name");
-        const char * prefix = plane.queryProp("@prefix");
-        StringBuffer path(prefix);
-        Owned<IEspTpDropZone> dropZone = createTpDropZone();
-        dropZone->setName(dropzonename);
-        dropZone->setDescription(""); //TODO
-        dropZone->setPath(path);
-        dropZone->setBuild("");
-        dropZone->setECLWatchVisible(plane.getPropBool("@eclWatchVisible", true));
-        StringBuffer localHost;
-        IArrayOf<IEspTpMachine> tpMachines;
-        Owned<IEspTpMachine> machine = createTpMachine();
-        IpAddress ipAddr;
-        ipAddr.ipset("localhost");
-        ipAddr.getIpText(localHost);
-        machine->setNetaddress(localHost.str());
-        machine->setConfigNetaddress("localhost");
-        machine->setDirectory(path);
-        machine->setOS(getPathSepChar(path) == '/' ? MachineOsLinux : MachineOsW2K);
-        tpMachines.append(*machine.getLink());
-        dropZone->setTpMachines(tpMachines);
-        list.append(*dropZone.getLink());
+        const char * path = plane.queryProp("@prefix");
+        if (!isEmptyString(path))
+        {
+            Owned<IEspTpDropZone> dropZone = createTpDropZone();
+            dropZone->setName(dropzonename);
+            dropZone->setDescription(""); //TODO
+            dropZone->setPath(path);
+            dropZone->setBuild("");
+            dropZone->setECLWatchVisible(plane.getPropBool("@eclWatchVisible", true));
```

Comment:
eclWatchVisible isn't part of the schema for a plane - doubt we want it to be.
I think for now, should always set to true here.